libs/design-system/navigation/table-of-contents/src/lib/table-of-contents/table-of-contents-item.component.html

```diff
@@ -0,0 +1,15 @@
+<nav class="link-container" [attr.aria-label]="section().label + ' Table of Contents'">
```

Comment:
The `<nav>` element should go in the table of content component and wrap all links